prereise/gather/demanddata/bldg_electrification/shapefiles/cb_2018_us_state_20m.prj

```diff
@@ -0,0 +1 @@
+GEOGCS["GCS_North_American_1983",DATUM["D_North_American_1983",SPHEROID["GRS_1980",6378137,298.257222101]],PRIMEM["Greenwich",0],UNIT["Degree",0.017453292519943295]]
```

Comment:
We are overlaying the state boundaries with the Balancing Authority boundaries from ba_area.shp to produce the map plots. I think its better to leave it in the repo, since overlaying is memory intensive